doc/releases/release-notes-2.4.rst

```diff
@@ -212,6 +212,14 @@ Drivers and Sensors
 
 * Ethernet
 
+  * Add VLAN support to Intel e1000 driver.
+  * Add Ethernet support to stm32h7 based boards.
+  * Add adds support for setting fixed configuration and read from device tree
```

Comment:
Maybe the "adds" is redundant? (not sure)

Reply:
> Maybe the "adds" is redundant? (not sure)

Just typo, will fix.